conf/local.conf.sample

```diff
@@ -1,10 +1,11 @@
-# This builds the iMX8M Plus by default 
```

Comment:
Do we need a default machine?  I thought we had to specify the machine when we configure Yocto.

Reply:
I used a  local.conf.sample included in our manifest as a template and that is their practice. To include a default machine.